week1/homework/src/commands.js

```diff
@@ -0,0 +1,25 @@
+
+function whatURL(command, state) {
+  switch (command) {
+    case 'state':
+      console.log('checked state')
```

Comment:
console.log are fine to use for cli applications, for when we really want to be informative, for example in the server when we say what port the server is listening on etc. If we use it for debugging purposes the prints should be removed when we are done debugging.